src/trossen_rslidar_main.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2024 Trossen Robotics - All Rights Reserved
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+
+#include <signal.h>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rs_driver/macro/version.hpp>
+#include <trossen_rslidar_node.hpp>
+
+// static void sigHandler(int sig)
+// {
+//   RS_MSG << "RoboSense-LiDAR-Driver is stopping....." << RS_REND;
+//   g_cv.notify_all();
+// }
+
+// void show_usage(std::string name)
+// {
+//   std::cerr << "Usage: " << name << " <option(s)> SOURCES"
+//             << "Options:\n"
+//             << "\t--config-path\t\tPass the config file absolute path\n"
+//             << std::endl;
+// }
+
+int main(int argc, char ** argv)
+{
+//   signal(SIGINT, sigHandler);  //< bind ctrl+c signal with the sigHandler function
+
+  RS_TITLE << "********************************************************" << RS_REND;
+  RS_TITLE << "**********    RSLidar_SDK Version: v" << RSLIDAR_VERSION_MAJOR
+           << "." << RSLIDAR_VERSION_MINOR
+           << "." << RSLIDAR_VERSION_PATCH << "     **********" << RS_REND;
+  RS_TITLE << "********************************************************" << RS_REND;
```

Comment:
@lukeschmitt-tr Do we want to retain prints like this? There is also a print command in the driver, which prints the configuration which the driver receives.

Reply:
You are right, we do not need these. Keeping the driver untouched is fine with me though.